tests/test_trainer.py

```diff
@@ -121,7 +121,7 @@ def test_mobilenet_v1_on_catdog_datasets_with_model_spec_override():
 def test_mobilenet_v1_on_catdog_datasets_with_num_gpus_override():
```

Comment:
This test was a bad idea anyway, training doesn't even run. I'd just delete it entirely (or maybe add in a check that `detect_num_gpus` is at least 2)